### PR TITLE
Symfony on Platform.sh docs update

### PR DESCRIPTION
**Pull request by fabpot**
This PR removes everything related to the old Symfony templates and redirects the user to the official docs for the Symfony integration.

### CODE REVIEW

File: docs/data/templates.yaml [hunk 1]
```diff
@@ -1845,35 +1845,6 @@ php:
       Redis 5.0<br />
       Automatic TLS certificates<br />
       Composer-based build<br />
-  - shortname: symfony4
```
Comment (fabpot) [inline docs/data/templates.yaml:1848]: Old deprecated template

File: docs/src/configuration/services/mysql/_index.md [hunk 2]
```diff
@@ -136,7 +136,6 @@ highlight=python
 ### Configure connections
 
 There may be cases where you want to configure a database connection manually.
-For example, when using a framework like Symfony Flex that expects the connection as an environment variable.
```
Comment (fabpot) [inline docs/src/configuration/services/mysql/_index.md:139]: As it's done automatically, the explanation below is not relevant for Symfony projects.

File: docs/src/development/email.md [hunk 3]
```diff
@@ -105,28 +105,6 @@ Beware of the potential security problems when using the `mail()` function,
 which arise when using user-supplied input in the fifth (`$additional_parameters`) argument.
 See the [PHP `mail()` documentation](http://php.net/manual/en/function.mail.php) for more information.
 
-### SwiftMailer
```
Comment (fabpot) [inline docs/src/development/email.md:108]: Swiftmailer is not maintained anymore and in any case, the docs below are only for Symfony, so not relevant anymore.

File: docs/src/languages/php/frameworks.md [hunk 4]
```diff
@@ -42,15 +42,9 @@ Template projects (repositories in the [platformsh-templates](https://github.com
...
-[Symfony](https://symfony.com/) is a web application framework written in PHP. Like Drupal, Symfony projects can utilize native Composer to build applications and manage dependencies.
+[Symfony](https://symfony.com/) is a web application framework written in PHP. Symfony projects utilize native Composer to build applications and manage dependencies.
 
-* [Symfony Best Practices](/frameworks/symfony/_index.md)
-
-### Community Support
-
-Symfony FAQs, how-to guides and other tutorials right on [Platform.sh Community](https://community.platform.sh/).
-
-* [Symfony on Platform.sh Community](https://community.platform.sh/search?expanded=true&q=symfony)
```
Comment (fabpot) [inline docs/src/languages/php/frameworks.md:53]: I didn't find anything specific to Symfony there, let's remove the link.

**Review by CollierCZ — APPROVED** [review]

**Pull request merged**